mvx-esdt-safe/tests/mvx_esdt_safe_blackbox_tests.rs

```diff
@@ -146,11 +150,13 @@ fn test_register_token_invalid_type() {
         MultiValueEncoded::from_iter(vec![token_hash]),
     );
 
+    let expected_logs =
+        vec![log!(REGISTER_TOKEN_ENDPOINT, topics: [DEPOSIT_EVENT], data: Some(INVALID_TYPE))];
```

Comment:
Everywhere where `DEPOSIT_EVENT` is expected, we should also check `EXECUTED_BRIDGE_OP_EVENT`.

Reply:
We can just keep this in mind for another PR where we can analyse the tests and set all expected_logs